### DISCUSSION TITLE
platform.js tries to clone `undefined`

### DISCUSSION THREAD

**Discussion by hyusetiawan** [vercel/next.js#13388] [Help]
# Bug report

## Describe the bug
platform.js JSON.stringify then JSON.parse (deep clone) `.default` property but that property is undefined. This only happens in dev but not in production (deployed on vercel)

## To Reproduce

Steps to reproduce the behavior, please provide code snippets or a repository:

I am not sure if there is an issue with my setup but it's just a simple page, called `flow.tsx`

## Expected behavior
no platform.js error and my page shows

## Screenshots
<img width="891" alt="Screen Shot 2020-05-26 at 2 09 14 AM" src="https://user-images.githubusercontent.com/1062920/82882439-e981de80-9ef5-11ea-8144-09fbeb2a3b0a.png">
look at the top right, shows undefined

## System information

- OS: macOS
- Browser: Chrome latest
- Version of Next.js: 9.4.2
- Version of Node.js: 12.16.0

## Additional context
here is my package.json: https://pastebin.com/ySaJLMGV
my next.config.js:
```
const withWorkers = require('@zeit/next-workers')
// TODO: use externals to minimize the size of the bundle: https://github.com/zeit/next.js/issues/7320
module.exports = withWorkers({
	/* config options here */
})
```
and my tsconfig.json: https://pastebin.com/feS103ZJ

I can load the index.tsx file just fine without the platform.js error, only flow.tsx page is causing an issue and it just returns an `<h3>flow page</h3>`

**Comment by hyusetiawan** [comment]
this seems to be caused by @next/react-dev-overlay 
I am honestly rather stuck, and my pages are all white screening, anyone has experienced this before?

**Comment by hyusetiawan** [comment]
I have to downgrade all the way to 9.3.6 to avoid the issue, is this a known problem in 9.4+?

**Comment by arwidt** [comment]
I have the same issue, 9.5.3.

**Comment by hyusetiawan** [comment]
this has been working if I use docker for some reason, I am really not sure why
would still love to get a resolution on this? because my docker's fs access is abysmal
